### DISCUSSION TITLE
Pre Proposal enquiry

### DISCUSSION THREAD

**Discussion by gluonparticle** [MDAnalysis/mdanalysis#4525] [GSoC Discussions]
@cbouy, @hmacdope, @xhgchen @orionarcher

Hi, I am Sargun.

I am really excited about the opportunity to make a contribution to the sector of open-source Quantum Chemistry by working on: "**Computational Framework for Simulating and Visualizing Complex Molecular Structures**". 

This is based on slight changes and some extra features from [the 4th project idea (Visualising atoms in 2D)](https://github.com/MDAnalysis/mdanalysis/wiki/GSoC-2024-Project-Ideas#project-4-2d-visualization-for-small-molecules) in the ideas list.

I have worked on a similar project in high school where I targeted on simulating [4f orbitals in 3D for Cerium atom](https://github.com/gluonparticle/3D-Orbitals).

the new features will include the addition of:

**- Showcasing electron clouds**
**- Probability of finding electrons based on bonds**  
**- Visualisation of complete molecules**
**- Visualising the reactions between atoms in real-time**
**- Consideration of other forces between two particles.**
**- Having a toggle option for viewing by orbitals or other models of atom**

The Github repository link of the old project: https://github.com/gluonparticle/3D-Orbitals ;

If you have any queries, suggestions for additional features, or would like me to modify my proposal in any way, please feel free to let me know. I'm open to feedback and eager to ensure that my contributions align with the goals and expectations of MDAnalysis. 

I also wanted to know where we can submit our pre-proposal form? 

Thank you for considering my software;

**Comment by orbeckst** [comment]
As I said in https://github.com/MDAnalysis/mdanalysis/discussions/4546#discussioncomment-8960228 , the deadline for the pre-proposal (see the blog post https://www.mdanalysis.org/2024/02/27/gsoc2024 ) is unfortunately already passed.

You're still welcome to contribute to MDAnalysis, of course.
